### PR TITLE
407 add timestep choice in config

### PR DESCRIPTION
**Pull request by marcochiaramello**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [x] main documentation was updated (update of input/output file)
- [ ] if this commit targets a release branch: the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

File: sources/Outputs/src/Solver.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  Par.cpp
```
Comment (rosiereflo) [inline sources/Outputs/src/Solver.cpp:12]: Wrong file name

File: documentation/functionalDoc/functionalDoc.tex [hunk 2]
```diff
@@ -232,10 +232,13 @@ \subsection{Static Var Compensators}
 \section{Solver}
 
 The solver used is the Simplified Solver (SolverSIM) from \Dynawo.
-The Maximum time step value is 10s.
+The Maximum time step value is by default 10s. This value can be modified in the configuration file (see section \ref{Dynaflow_Launcher_Configuration_Configuration_File}).
```
Comment (rosiereflo) [inline documentation/functionalDoc/functionalDoc.tex:235]: Maybe we should specify that the value provided should be above 1 (hMin)

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: sources/Outputs/src/Solver.cpp (hunk 1 above)
Comment (marcochiaramello) [reply sources/Outputs/src/Solver.cpp:12]: thx

File: documentation/functionalDoc/functionalDoc.tex (hunk 2 above)
Comment (marcochiaramello) [reply documentation/functionalDoc/functionalDoc.tex:235]: done, thx

**Pull request merged**